src/openapi/transport/batch.js

```diff
@@ -67,6 +80,7 @@ function batchCallSuccess(callList, batchResult) {
  * @param {Array.<{method: string, args:Array}>} callList
  */
 function runBatchCall(serviceGroup, callList) {
+    const parentRequestId = getRequestId();

```

Comment:
Request id for container request that contains all child batched requests.
It's need to be requested before child requests are build to ensure that parent id has lower value then first child 
request. This ensures easier mapping to local ids when parsing response.